scss/detail.scss

```diff
@@ -0,0 +1,85 @@
+.book-title {
+  height: 39px;
+  width: 210px;
+  color: #000;
+  font-family: Roboto;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38px;
+  margin: 10px 0 40px 10px;
+}
+
+.book-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 130px;
+  margin: 0 0 0 10px;
```

Comment:
```suggestion
  margin-bottom: 10px;
```

Reply:
@CrisVega08 `margin-left`?

Reply:
I assumed he wanted to say left, yes! :+1:

Reply:
It really is `margin-left`